app/config/image_variations.yml

```diff
@@ -20,3 +20,9 @@ ezpublish:
                     reference: ~
                     filters:
                         - { name: geometry/scalewidthdownonly, params: [710] }
+liip_imagine:
+    driver: gd
+    filter_sets:
+        blog_thumbnail:
```

Comment:
Indeed, it works in the same way. I was thinking that it will be easier to read and change if everything is grouped.
What do you think @damianz5 ?